.cirrus.yml

```diff
@@ -367,7 +372,7 @@ image_prune_task:
 
 # This task does the unit and integration testing for every platform
 testing_task:
-
+    alias: "testing"
```

Comment:
Oh neat! Didn't know about that feature.  It's a bit hidden in the docs :confused:

Reply:
Oh!  Beware:  Using Alias will impact `contrib/cirrus/cirrus_yaml_test.py` in cases where the task name and alias name differ.  Should be easy to fix though.